client/Dockerfile-client

```diff
@@ -0,0 +1,40 @@
+# Our First stage, that builds the application
+FROM node:12.13.0-alpine AS react-builder
```

Comment:
It is recommended to use images from `helsinkitest` (https://hub.docker.com/u/helsinkitest) as a base. `helsinkitest/node:12-slim` image should be appropriate in this case.